forms-flow-web/src/styles.scss

```diff
@@ -49,22 +13,22 @@ $dark: $gray-800;
 }
 
 .text-primary {
-  color: $primary;
+  color: var(--color-info);
 }
```

Comment:
I have doubt that text-primary class already defined on MF with var(--color-primary) i think.so this one does make any sense

Reply:
good catch.

Reply:
yes. text-primary class need to bring inside theme service/bootstrapOverride and need to use color-primary

Reply:
good catch @fahad-aot 
The .text-primary class is already declared in Global.scss file and that will override this class and as you said it doesn't make any impact. 
Need to remove all the unwanted redundant classes from styles.scss, may be with another ticket.

Reply:
okay.